### ISSUE TITLE
Found mach-o universal

### ISSUE THREAD

**Issue by iOS-Nishant** [Maximus-/class-dump-swift#5]
Hi Max,
I tried running swiftd on a valid test binary and got following message:
`Found mach-o universal 0x7f9353500000`
![swiftd](https://cloud.githubusercontent.com/assets/6300828/22924711/2d90a4dc-f2cc-11e6-8357-baeb03a147a0.png)

**Comment by Maximus-** [comment]
I currently don't support universal mach-o's. I'll be fixing this when I get some time in the coming weeks.

**Comment by kryptoknight13** [comment]
Hi @Maximus- Even I am also facing the above issue.

**Comment by Maximus-** [comment]
@kryptoknight13 It's known. Will be fixed when I get time to refactor it.

**Comment by rustymagnet3000** [comment]
Yeah, worked for me when I specifed that xCode build for a single target (ARM7) instead of pumping out a universal binary so it could support device simulators.

@Maximus- cool work!

**Comment by alokc83** [comment]
you can also use lipo to extract out specific arch to analyze
